### PR TITLE
Hideout: Shooting Range - added pack of screws

### PR DESCRIPTION
**Pull request by ckaram91**
[Discord Post: #bugs-data](https://discord.com/channels/668387296973684737/668387739808563221/921320294067605524)

image

They added requirement for two packs of screws for Shooting Range.

### CODE REVIEW

File: hideout.json [hunk 1]
```diff
@@ -1754,6 +1754,12 @@
                     "name": "57347c1124597737fb1379e3",
                     "quantity": 3,
                     "id": 134
+                },
+                {
+                    "type": "item",
+                    "name": "59e35ef086f7741777737012",
+                    "quantity": 2,
+                    "id": 313
```
Comment (ckaram91) [inline hideout.json:1762]: Item reference: https://github.com/ckaram91/tarkovdata/blob/hideout/shooting-range/items.en.json#L3167-L3170

File: hideout.json [hunk 2]
```diff
@@ -1754,6 +1754,12 @@
                     "name": "57347c1124597737fb1379e3",
                     "quantity": 3,
                     "id": 134
+                },
+                {
+                    "type": "item",
+                    "name": "59e35ef086f7741777737012",
```
Comment (ckaram91) [inline hideout.json:1760]: Item reference: https://github.com/ckaram91/tarkovdata/blob/hideout/shooting-range/items.en.json#L3167-L3170

File: hideout.json (hunk 1 above)
Comment (ckaram91) [inline hideout.json:1762]: Increased id here as well, so 312 doesn't exist yet in this branch:

https://github.com/TarkovTracker/tarkovdata/pull/173/files#diff-fdf986595135ba1fa4f4c6af72d5cdef425a35015ca795fc6745d54d4956c97cR1806

**Review by kokarn — APPROVED** [review]
LGTM

**Pull request merged**